dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -251,6 +254,15 @@ DataInterfaceIIDM::getBusName(const std::string& componentName, const std::strin
   return busName;
 }
 
+std::vector<stdcxx::Reference<powsybl::iidm::ThreeWindingsTransformer::Leg> >
+DataInterfaceIIDM::getLegs(powsybl::iidm::ThreeWindingsTransformer& ThreeWindingsTransformerIIDM) {
```

Comment:
since this is used just once maybe we can put this inside the convert method?

Reply:
I liked this way to make convert method even lighter

Reply:
I would propose something else: create a method that handles one leg (with the leg and an index as parameters) and call it on the three legs in the main method